src/test/java/org/folio/circulation/services/HealthServiceTest.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.services;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import org.folio.circulation.CirculationVerticle;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(VertxExtension.class)
+class HealthServiceTest {
+
+  @Test
+  void health(Vertx vertx, VertxTestContext vtc) {
+    var options = new DeploymentOptions().setConfig(new JsonObject().put("port", 8081));
```

Comment:
Port 8081 for this test? Many modules started by "hand" uses this port, so I'd rather you use 9230-9239 or a random port.